src/transformers/models/marian/convert_marian_tatoeba_to_pytorch.py

```diff
@@ -46,194 +47,236 @@ class TatoebaConverter:
 
     Steps:
 
-        1. convert numpy state dict to hf format (same code as OPUS-MT-Train conversion).
-        2. rename opus model to huggingface format. This means replace each alpha3 code with an alpha2 code if a unique
+        1. Convert numpy state dict to hf format (same code as OPUS-MT-Train conversion).
+        2. Rename opus model to huggingface format. This means replace each alpha3 code with an alpha2 code if a unique
            one exists. e.g. aav-eng -> aav-en, heb-eng -> he-en
-        3. write a model card containing the original Tatoeba-Challenge/README.md and extra info about alpha3 group
-           members.
+        3. Select the best model for a particular pair, parse the yml for it and write a model card. By default the
+           best model is the one listed first in released-model-results, but it's also possible to specify the most
+           recent one.
```

Comment:
it seems the script needs to be run from inside the Transformers repo, could maybe add a note about that here.